src/pages/javascript/ifelse-statements/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: The JavaScript Ifelse Statement
+title: Ifelse Statement
```

Comment:
Can you change the title to `If-else Statement` and the path to `if-else-statement`?